cmd/server/storage/db/get_data.go

```diff
@@ -0,0 +1,86 @@
+package db
+
+import (
+	"context"
+
+	"github.com/ramil063/gometrics/cmd/server/storage/db/dml"
+	"github.com/ramil063/gometrics/internal/logger"
+	"github.com/ramil063/gometrics/internal/models"
+)
+
+func (s *Storage) GetGauge(name string) (float64, bool) {
```

Comment:
```suggestion
func (s *Storage) GetGauge(name string) (float64, error) {
```
Всегда необходимо возвращать ошибку, а не факт ее наличия или отсутствия. Кроме того, в случае со Scan необходимо анализировать тип ошибки, так как Scan генерит ошибку в том числе в случае ненахождения искомой записи.